MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include config/slots.json
```

Comment:
Is this file used by `setup.py`, or is an artifact from your local build process?

Reply:
Currently the package has only been cloned, so the setup is never run. If you run the setup, the package directly will only contain the files setuptools recognizes as necessary for the package to function, which _excludes_ data files. The `setup.py` up until now used `data_files`, but that is deprecated (and doesn't work with `wheels`). 

The `setup.py` also has the argument `include_package_data=True`, which checks the `MANIFEST.in` file for any files that should be included in the package directory when installed. However, there was no manifest file to check. Hence, I created a `MANIFEST.in` file that includes the data files, and removed the deprecated `data_files` argument.

Reply:
Thanks for the explanation. This is another example of a change that is self-contained and should be its own commit. And the description you provided above would be in the commit message.

Again, no need to change this one, but it should be kept in mind for future commits.